etc/ssokenizer.yml

```diff
@@ -38,6 +38,48 @@ identity_providers:
     # variables easier.
     scopes:
       - "$GOOGLE_SCOPES"
+      
+  github:
+    # amazon, bitbucket, facebook, github, gitlab, google, heroku, microsoft,
```

Comment:
Let's remove all the comments from the `github`/`heroku` sections. I put these in the `google` section so people looking at the repo would know what they do, but it's redundant to spell that out for each provider.